src/content/operations/cleanup/_index.en.md

```diff
@@ -0,0 +1,144 @@
+---
+title: "Uninstalling Submariner"
+date: 2020-12-23T21:25:11+01:00
+weight: 50
+---
+
+To properly uninstall Submariner from a cluster, follow the steps below:
+
+{{% notice note %}}
+Make sure KUBECONFIG for all participating clusters is exported and all participating clusters are accessible via kubectl.
+{{% /notice %}}
+
+1. Delete Submariner-related namespaces
+
+   For each participating cluster, issue the following command:
+
+   ```bash
+   kubectl delete namespace submariner-operator
+   ```
+
+   For the Broker cluster, issue the following command:
+
+   ```bash
+   kubectl delete namespace submariner-k8s-broker
```

Comment:
With subctl-devel (0.9 and up), it is also required to delete submariner-operator namespace from the broker cluster:
see: https://github.com/submariner-io/submariner-operator/issues/203

Reply:
Correct. Added that section.